websites/V/VidCloud9/dist/metadata.json

```diff
@@ -18,6 +19,7 @@
   ],
   "category": "videos",
   "iframe": true,
+  "readLogs": false,
   "settings": [
```

Comment:
no need to set to false since thats the default value

Reply:
okay then I'll change that